src/poliastro/twobody/orbit.py

```diff
@@ -1094,9 +1095,13 @@ def plot(self, label=None, use_3d=False):
             Label for the orbit, defaults to empty.
         use_3d : bool, optional
             Produce a 3D plot, default to False.
-
+        static : bool, optional
+            Produce a static image of the figure, default to false
         """
-        if use_3d:
+        from poliastro.plotting.static import StaticOrbitPlotter
```

Comment:
I understand that this is here because of circular imports. What's the conflict exactly?

Reply:
Oh, sorry, I see it here https://github.com/poliastro/poliastro/pull/639#issuecomment-479574268